js_modules/dg-docs/components/lib/PackageDetails.tsx

```diff
@@ -0,0 +1,27 @@
+import {Fragment, ReactNode} from 'react';
+
+import styles from './css/PackageDetails.module.css';
+
+import ComponentHeader from './ComponentHeader';
+import {DocsLinkProps, Package} from './types';
+
+interface Props {
+  pkg: Package;
+  renderLink: (props: DocsLinkProps) => ReactNode;
```

Comment:
This prop allows the caller to render the desired Link component. In `site`, this is `next/link`. In `ui-core`, it will be the `react-router-dom` link component.